packages/insured-bridge-relayer/src/Relayer.ts

```diff
@@ -496,23 +499,12 @@ export class Relayer {
     };
   }
 
-  private async _disputeRelay(deposit: Deposit, relay: Relay) {
-    // We shouldn't be batching disputes because we don't expect to send batches of them.
-    const { receipt } = await this._sendTransaction(
-      this._generateDisputeRelayTx(deposit, relay),
-      "Disputed pending relay. Relay was deleted 🚓",
-      this._generateMrkdwnForDispute(deposit, relay)
-    );
-
-    if (receipt?.events?.RelayCanceled)
-      this.logger.error({
-        at: "AcrossRelayer#Disputer",
-        message: "Dispute failed to send to OO 👀!",
-        tx: receipt.transactionHash,
-        depositHash: receipt.events.RelayCanceled.returnValues.depositHash,
-        relayHash: receipt.events.RelayCanceled.returnValues.relayHash,
-        disputer: receipt.events.RelayCanceled.returnValues.disputer,
-      });
+  private async _generateDisputeRelayTransaction(deposit: Deposit, relay: Relay) {
```

Comment:
this method now returns the dispute transaction, just like the similar methods in this file `_generateRelayTransaction` and `_generateSettleTransactionForSettleableRelay`